src/dawn/CodeGen/GridTools/GTCodeGen.cpp

```diff
@@ -91,11 +99,6 @@ GTCodeGen::IntervalDefinitions::IntervalDefinitions(const Stencil& stencil)
     for(const Interval& interval : DoMethodIntervals)
       IntervalToNameMap.emplace(interval, Interval::makeCodeGenName(interval));
   }
-
-  // Make sure the intervals for the stencil functions exist
-  for(const auto& stencilFun : stencil.getStencilInstantiation().getStencilFunctionInstantiations())
-    IntervalToNameMap.emplace(stencilFun->getInterval(),
-                              Interval::makeCodeGenName(stencilFun->getInterval()));
```

Comment:
why do we no longer need this?

Reply:
because it is done in line 82. It was very redundant, we were iterating over stencil function instantiations to insert into stencils and then again to insert into intervalToNameMap. Now we just insert into stencils, and from them we build the intervalToNameMap

Reply:
ok